go install error

```
➜  ~ go get -d github.com/bad33ndj3/ynab-lazy-import && go install github.com/bad33ndj3/ynab-lazy-import 
# github.com/bad33ndj3/ynab-lazy-import/cmd
go/src/github.com/bad33ndj3/ynab-lazy-import/cmd/init.go:74:51: not enough arguments in call to c.ynab.Account().GetAccounts
	have (string)
	want (string, *api.Filter)
go/src/github.com/bad33ndj3/ynab-lazy-import/cmd/init.go:79:15: cannot range over accountList (type *account.SearchResultSnapshot)
```

 github.com/brunomvsouza/ynab.go doesnt implement v2 correct

--- a/README.md
+++ b/README.md
@@ -10,9 +10,9 @@ This way the upload will behave just like a CSV Upload to YNAB.
 This approach is used because ING doesn't offer a consumer based API.
 
 ### Usage
-Install package
+Install package trough [gobinaries.com](https://gobinaries.com/)
 
-    $ go get -d github.com/bad33ndj3/ynab-lazy-import && go install github.com/bad33ndj3/ynab-lazy-import
+    $ curl -sf https://gobinaries.com/bad33ndj3/ynab-lazy-import | sh
 
 Initialize the config file and fill in the missing fields
 

